### ISSUE TITLE
A11y_Azure Tools Storage Explorer_ Manage Access policies_ Keyboard: Keyboard focus is not going to "Learn more" link in the "Manage Access policies" dialog box.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/AzureStorageExplorer#4976]
### GitHubTags:
#A11y_Azure Tools Storage Explorer; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Azure Tools Storage Explorer; #MAC; #BM-AzureToolsStorageExplorer-MAC-Sep2021; #WCAG2.1.1; #Keyboard; #AINotInScope;#A11yMAS;

### Environment Details:​
Application Name: Azure Tools Storage Explorer
Standard Violated: Keyboard

### Repro Steps:​
1.Open Azure Tools Storage Explorer 
2.Tab till the "Blob containers" and press Enter.
3.Right click on any existing blob container.
4.Tab till "Manage Access policies" and press enter.
5.Verify whether the keyboard focus is going to "Learn more about access policy permissions" link or not.
### Actual Result:
keyboard focus is not going to "Learn more about access policy permissions" link in "Manage Access policies" dialog box .

### Expected Result: ​
keyboard focus  should  go to "Learn more about access policy permissions" link in "Manage Access policies" dialog box .

### User Impact:
Keyboard users will find difficult in finding the "Learn more about access policy permissions" link if the keyboard focus is not going to "Learn more about access policy permissions" link in "Manage Access policies" dialog box .

### Attachments:
![Focus is not going to link](https://user-images.githubusercontent.com/89508906/135066829-a4912e32-ef33-4016-bab0-1f4897124b3a.gif)

**Comment by MRayermannMSFT** [comment]
Related https://github.com/microsoft/AzureStorageExplorer/issues/4977

**Comment by Latha2564** [comment]
<img width="270" alt="Screenshot 2021-11-03 at 11 34 00 AM" src="https://user-images.githubusercontent.com/89508906/140015891-e2c6ce4e-ac01-475b-a6d7-46652b5c0b3e.png">

**Comment by Latha2564** [comment]
The Bug is repro in latest Environment with  the same build number mentioned in the Screenshot.

**Comment by Latha2564** [comment]
The focus is going to the "learn More " link in he "Manage Access policies dialog box.Hence closing the bug.
![Keyboard focus going to the Learn more link](https://user-images.githubusercontent.com/89508906/141959900-cd2662d2-b7ec-4cb8-92c6-5bd3cac92db2.gif)

**Comment by Sobi261** [comment]
#Closed;
